scripts/dev/update_3rdparty.py

```diff
@@ -104,8 +104,7 @@ def update_pdfjs(target_version=None, legacy=False, gh_token=None):
     else:
         # We need target_version as x.y.z, without the 'v' prefix, though the
         # user might give it on the command line
-        if target_version.startswith('v'):
-            target_version = target_version[1:]
+        target_version = target_version.removeprefix('v')
```

Comment:
`str.removeprefix()` [does nothing](https://docs.python.org/3.9/library/stdtypes.html#str.removeprefix) if there is no such prefix.